ethstore/src/ethstore.rs

```diff
@@ -244,11 +245,23 @@ impl EthMultiStore {
 			vaults: Mutex::new(HashMap::new()),
 			iterations: iterations,
 			cache: Default::default(),
+			dir_hash: Default::default(),
 		};
 		store.reload_accounts()?;
 		Ok(store)
 	}
 
+	fn reload_if_changed(&self) -> Result<(), Error> {
+		let mut last_dir_hash = self.dir_hash.lock();
```

Comment:
@svyatonik 
yeah but i guess users are not supposed to mess with the vault directories themselves